builtin/locale/__builtin.ru.tr

```diff
@@ -0,0 +1,246 @@
+Unhandled remove_player return code @1.=Необработанный код возврата remove_player @1.
+Cannot teleport out of map bounds!=Невозможно телепортироваться за пределы карты!
+Cannot get player with name @1.=Не удается получить игрока с именем @1.
+Cannot teleport, @1 is attached to an object!=Невозможно телепортироваться, @1 привязан к объекту!
+Teleporting @1 to @2.=Телепортация @1 в @2.
+One does not teleport to oneself.=Игрок не может телепортироваться к самому себе.
+Cannot get teleportee with name @1.=Не удается найти телепортирующегося с именем @1.
+Cannot get target player with name @1.=Не удается найти целевого игрока с именем @1.
+Teleporting @1 to @2 at @3.=Телепортация @1 к @2 в @3.
+<X>,<Y>,<Z> | <to_name> | <name> <X>,<Y>,<Z> | <name> <to_name>=<X>,<Y>,<Z> | <to_name> | <имя> <X>,<Y>,<Z> | <имя> <имя_к_кому>
+Teleport to position or player=Телепортироваться на позицию или к игроку
+You don't have permission to teleport other players (missing privilege: @1).=У вас нет разрешения на телепортацию других игроков (отсутствует привилегия: @1).
+([-n] <name> <value>) | <name>=([-n] <имя> <значение>) | <имя>
+Set or read server configuration setting=Установить или прочитать настройки конфигурации сервера
+Failed. Cannot modify secure settings. Edit the settings file manually.=Ошибка. Не удается изменить настройки безопасности. Отредактируйте файл настроек вручную.
+Failed. Use '/set -n <name> <value>' to create a new setting.=Ошибка. Используйте '/set -n <имя> <значение>', чтобы создать новую настройку.
+@1 @= @2=@1 @= @2
+<not set>=<не установлена>
+Invalid parameters (see /help set).=Недопустимые параметры (см. /help set).
+Finished emerging @1 blocks in @2ms.=Загрузка/генерация @1 map-блоков завершена за @2мс.
+emergeblocks update: @1/@2 blocks emerged (@3%)=обновление emergeblocks: загружено/сгенерировано @1/@2 блока (@3%)
+(here [<radius>]) | (<pos1> <pos2>)=(here [<радиус>]) | (<pos1> <pos2>)
+Load (or, if nonexistent, generate) map blocks contained in area pos1 to pos2 (<pos1> and <pos2> must be in parentheses)=Загрузить (или, если не существует, сгенерировать) map-блоки, содержащиеся в области с pos1 по pos2 (<pos1> и <pos2> должны быть в круглых скобках)
+Started emerge of area ranging from @1 to @2.=Загрузка/генерация области в диапазоне от @1 до @2 начата.
+Delete map blocks contained in area pos1 to pos2 (<pos1> and <pos2> must be in parentheses)=Удалить map-блоки, содержащиеся в области с pos1 по pos2 (<pos1> и <pos2> должны быть заключены в круглые скобки)
+Successfully cleared area ranging from @1 to @2.=Успешно очищена область в диапазоне от @1 до @2.
+Failed to clear one or more blocks in area.=Не удалось очистить один или несколько блоков в области.
+Resets lighting in the area between pos1 and pos2 (<pos1> and <pos2> must be in parentheses)=Сбрасывает освещение в области между позициями 1 и pos2 (<pos1> и <pos2> должны быть заключены в круглые скобки)
```

Comment:
You may translate `<pos1>` and `<pos2>` (in case you missed it).

Reply:
> You may translate `<pos1>` and `<pos2>` (in case you missed it).


I think `с позиции1 по позицию2` will read worse than if we remain english `pos1` and `pos2` in this case. Also it will increase the length of an already long sentence.